L1Trigger/L1THGCal/interface/backend/HGCalHistoSeedingImpl.h

```diff
@@ -151,6 +151,7 @@ class HGCalHistoSeedingImpl {
   unsigned nBins2_ = 216;
   std::vector<unsigned> binsSumsHisto_;
   double histoThreshold_ = 20.;
+  double area_per_triggercell_ = 4.9115082E-05;
```

Comment:
- To be defined as `static constexpr`. There is no reason to make it configurable.
- The precise value of this number is probably not important. I would reduce the number of digits (maybe `4.91` would be enough)